toxics/slicer.go

```diff
@@ -37,10 +37,13 @@ func (t *SlicerToxic) chunk(start int, end int) []int {
 		return []int{start, end}
 	}
 
+	mid := start + (end-start)/2
 	// +1 in the size variation to offset favoring of smaller
 	// numbers by integer division
 	//#nosec
-	mid := start + (end-start)/2 + (rand.Intn(t.SizeVariation*2) - t.SizeVariation) + rand.Intn(1)
+	if t.SizeVariation > 0 {
+		mid = mid + (rand.Intn(t.SizeVariation*2) - t.SizeVariation) + rand.Intn(2)
```

Comment:
```suggestion
		mid = mid + rand.Intn(t.SizeVariation*2) - t.SizeVariation
```

Reply:
I propose keeping the adjustment inside the `if`. Doing so has the nice property of making the slicing deterministic if `SizeVariation=0`.

With the code now, if someone has `SizeVariation=0`, they are guaranteed to receive their data in `NumberOfBytes / AverageSize` equally sized chunks (plus 1 if there is a remainder).

Reply:
The part `rand.Intn(2)` is connected to  to `(end-start)/2`. Other proposal is to remove `rand.Intn(2)`, as it was before behavior.  I think `rand.Intn(t.SizeVariation*2)` is enough for randomization.